app/models/qualification.rb

```diff
@@ -21,4 +26,24 @@ def remove_irrelevant_data
       self.year = nil
     end
   end
+
+  def finished_studying_option_with_details
```

Comment:
This feels like it probably shouldn't be in the model

Reply:
Agreed. Presenter?

Reply:
Hmmm. I still don't think we've ironed out presenter/viewcomponent/helper (s) and how we want to use each of them. 

Do you want to drop a message in the engineering chat about it?

Reply:
Got rid of method